cluster-scope/overlays/prod/moc/zero/nodenetworkconfigurationpolicies/moc-nfs-network.yaml

```diff
@@ -3,7 +3,6 @@ kind: NodeNetworkConfigurationPolicy
 metadata:
   name: moc-nfs-network-enp4s0f1np1
 spec:
-  parallel: true
```

Comment:
We were setting `parallel: true` explicitly to work around [a problem][1] (see a37d254a2d). I suspect it's safe to remove, although I'd like to follow up with someone on the networking side to see if this is actually a bug in the openapi spec.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1967771

Reply:
Tried tests again just to be sure and it really gives an error (on picture below) which might suggest a bug in the openapi spec... Anyway these changes are done so that we don't have to delete whole zero (because rest of manifest seem fine), so it shouldn't matter too much anyway.
![image](https://user-images.githubusercontent.com/47832967/144819033-b819067f-ef27-442e-941c-a47519aaf73e.png)